src/components/shared/IPBlocker.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactNode } from "react";
+import IPData from "ipdata";
+import Tenant from "@/lib/tenant/tenant";
+import Image from "next/image";
+import { Button } from "@/components/ui/button";
+
+// Cuba, Iran, North Korea, Syria, and Russia are blocked from participating in the airdrop
+const BLOCKED_COUNTRIES = ["CU", "IR", "KP", "SY", "RU"];
+
+// The following ISO 3166-2 codes are for Crimea, Donetsk and Luhansk regions of occupied Ukraine
+// https://www.iso.org/obp/ui/#iso:code:3166:UA
+const BLOCKED_REGIONS = ["UA-43", "UA-14", "UA-09"];
```

Comment:
do you think we would want to make this tenant specific? Probably fine for now since scroll is the only airdrop we are interested in gating now.